### DISCUSSION TITLE
C++ Support?

### DISCUSSION THREAD

**Discussion by loickal** [supabase/supabase#1603] [Questions] [answered]
Hello folks

I'm currently learning C++ and saw supabase a few weeks ago in fireships yt video.
I wanted to ask you guys if anyone knows if and how I could use Supabase authentication in C++?

Thanks for the answers

Best regards
Loïc Kalbermatter

**Comment by MartyMcFlyInTheSky** [comment]
We're currently working on a c++ client library. But it is not an official one.

**Comment by J0** [comment, marked as answer]
Supabase Engineer here. We don't have an official C++ library at the moment -- most of the community libs are housed in the [Supabase-community Org](https://github.com/supabase-community)

Don't think we're working on a C++ lib at the moment but we do take contributions to the community repo

**Comment by meryemElkorchi** [comment]
Hello ;
I'm currently working in a C++ application and I want to do supabase authentication using the REST API but I haven't figured out how. I wanted to ask  if anyone knows  how I could use Supabase authentication in C++?
 Thanks for the answers

Best regards
